src/components/Dashboard/TodoList.vue

```diff
@@ -251,18 +223,19 @@
                       </v-btn>
                     </template>
                     <v-list class="actions__more-actions">
-                      <v-list-item
-                        v-if="businessId"
-                        id="btn-draft-delete"
-                        @click.stop="confirmDeleteDraft(item)"
-                      >
+                      <v-list-item id="btn-draft-delete" @click.stop="confirmDeleteDraft(item)">
                         <v-icon class="pr-1" color="primary" size="18px">mdi-delete-forever</v-icon>
-                        <v-list-item-title>Delete changes to company information</v-list-item-title>
+                        <v-list-item-title>Delete draft</v-list-item-title>
                       </v-list-item>
                     </v-list>
                   </v-menu>
                 </template>
 
+                <!-- other correction or conversion -->
+                <template v-else-if="isTypeCorrection(item) || isTypeConversion(item)">
+                  <!-- no action button in this case -->
+                </template>
```

Comment:
Corrections and conversion don't have actions for regular users. For staff, the buttons are displayed in section above (line 200+).

Reply:
Screenshot (staff):

![image](https://user-images.githubusercontent.com/10002889/207436212-7a00d5eb-0c66-4589-a6ae-3fd6977dc546.png)